discoveries/coastal-flooding-and-slr.discoveries.mdx

```diff
@@ -0,0 +1,150 @@
+---
+featuredOn:
+  - eis
+id: 'eis-coastal-risks'
+name: Risks to coasts from sea level rise compounded by water cycle intensification
+description: "NASA measurements and models shed light on present and future coastal risks"
+media:
+  src: ::file ./coastal-flooding-and-slr-cover.jpg
+  alt: Flooding in York, UK - February 2020.
+  author:
+    name: Unknown
```

Comment:
The Unsplash page lists the author as `Don Lodge`. Is Unknown intentional?

Reply:
Not intentional. I didn't see the author's name on that page. Thanks, @danielfdsilva!